src/main/java/com/google/sps/data/GoogleShoppingResultsWrapper.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.io.IOException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+/**
+ * Return the Google Shopping results for a given query in the form of an HTML page. 
+ */
+public class GoogleShoppingResultsWrapper {
+
+  private static final String GOOGLE_SEARCH_URL = "https://www.google.com//search";
+  private static final String searchType = "tbm=shop";
```

Comment:
If a variable stores constant value, then the convention is to capitalize every letter. 
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

I think  you don't need to have three variables here and always concatenate them in the method. You can put all of them in the BASE URL and add comments above to explain those parameters.

Reply:
Thanks for letting me know.
And like you suggested I'll add all the variables to the Base Url directly.
I though it'll be more readable if I added the variables separately, but I think its not a good idea in this case :E